assets/js/views/Msg.coffee

```diff
@@ -0,0 +1,27 @@
+define (require) ->
+  _BaseViews = require '_BaseViews'
+  tpl = require 'raw!../../templates/msg.html'
+
+  class SimulatorAddView extends _BaseViews.View
+    className: 'connection-error'

```

Comment:
Össze van itt keveredve ez-az, SimulatorAddView, connection-error class (mikor lehet success message is)